changelog/util.go

```diff
@@ -0,0 +1,27 @@
+package changelog
```

Comment:
Good catch, and you're not missing anything. I elected to copy code in this PR to keep changes small. The work to refactor the `actions/changelog_util.go` and the global changelog generation code to use this new changelog package is captured in https://github.com/deis/deisrel/issues/51, and I'm going to pick that up right after this PR is merged